app/controllers/submit.js

```diff
@@ -556,22 +604,51 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             let currentAbstract = node.get('description');
             let currentTags = node.get('tags').slice(0);
             let currentDOI = model.get('doi');
+            let currentLicenseType = model.get('license');
+            let currentLicenseRecord = model.get('licenseRecord');
+            let currentNodeLicenseType = node.get('license');
+            let currentNodeLicenseRecord = node.get('nodeLicense');
+            let newCopyrightHolders = [''];
+            if (this.get('basicsLicense.copyrightHolders') && this.get('basicsLicense.copyrightHolders').length) {
+                newCopyrightHolders = this.get('basicsLicense.copyrightHolders').split(',');
```

Comment:
I believe that is intended behavior. It allows previous selections to be cached if the license ends up changing to one that requires those fields. Those are also not displayed when viewing the preprint's license if they are not required.